aip/auth/4110.md

```diff
@@ -113,8 +113,8 @@ digraph d_front_back {
   execute [ label="5. Execute Auth Flows" ];
   post_processing [ label="6. Post Processing" ];
 
-  check_env_var -> load_credentials -> check_metadata ->auth_flows -> execute -> post_processing;
-  load_credentials -> auth_flows;
```

Comment:
I would keep the picture the way it was and add the new step. It would better represent what is happening. 

Our steps are messed up a bit, we better fix them later than mess up the picture as well )